python/pyspark/sql/connect/udf.py

```diff
@@ -74,22 +72,22 @@ def _create_py_udf(
     else:
         is_arrow_enabled = useArrow
 
-    regular_udf = _create_udf(f, returnType, PythonEvalType.SQL_BATCHED_UDF)
-    try:
-        is_func_with_args = len(getfullargspec(f).args) > 0
-    except TypeError:
-        is_func_with_args = False
+    eva_type: int = PythonEvalType.SQL_BATCHED_UDF
```

Comment:
Just wanted to confirm if `eva_type` is typo or intentional : )

Reply:
It's definitely a typo. :)